samples/captioning/multi-track-captions.html

```diff
@@ -19,22 +19,76 @@
             controlbar,
             player;
 
-        function startVideo() {
-            var url = FRAGMENTED_CAPTION_URL,
-                TTMLRenderingDiv;
-
+        function init() {
+            var TTMLRenderingDiv = document.querySelector("#ttml-rendering-div");
             videoElement = document.querySelector(".videoContainer video");
-            TTMLRenderingDiv = document.querySelector("#ttml-rendering-div");
-
             player = dashjs.MediaPlayer().create();
-            player.initialize(videoElement, url, true);
-            player.setTextDefaultEnabled(true);
-            player.setInitialMediaSettingsFor('fragmentedText', { 
-                lang: 'swe'
-            });
+            player.initialize(videoElement);
             player.attachTTMLRenderingDiv(TTMLRenderingDiv);
+            player.on('currentTrackChanged', function(e) { 
+                if (e.newMediaInfo.type === 'fragmentedText') {
+                    setTimeout(showDomStorage, 0);
+                }
+            });
             controlbar = new ControlBar(player); // Checkout ControlBar.js for more info on how to target/add text tracks to UI
```

Comment:
Can we please use /* ... */ for comments, otherwise the code on the sample page looks like a comment